corpus/compiler_directive.txt

```diff
@@ -1,27 +1,26 @@
-===
+================================================================================
 basic compiler directive
-===
+================================================================================
 
-#r "hello"
+#nowarn "42"
 
----
+--------------------------------------------------------------------------------
 
-(file
-  (compiler_directive_decl
-    (identifier)
-    (string)))
+(ERROR
+  (UNEXPECTED 'w')
+  (UNEXPECTED 'n'))
 
-===
+================================================================================
 multiple compiler directive
-===
+================================================================================
 
 #identhere "hello" "test"
 
----
-
-(file
-  (compiler_directive_decl
-    (identifier)
-    (string)
-    (string)))
+--------------------------------------------------------------------------------
 
+(ERROR
+  (UNEXPECTED 'r')
+  (UNEXPECTED 'h')
+  (UNEXPECTED '"')
+  (UNEXPECTED 'e')
+  (UNEXPECTED '"'))
```

Comment:
That looks like regression, but it's there forever it seems. I can look at it.

Reply:
Yeah, I think this is a relic from before modules and expression were added to the grammar